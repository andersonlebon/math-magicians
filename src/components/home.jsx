import React from 'react';

const Home = () => (
  <section className="p-5">
    <h2>Welcome to our page</h2>
    <p className="px-4 py-2">
      Math is beautiful. But sometimes, this can be hard to see, and even harder
      to convey to students who don’t — yet — share your passion. In addition to
      math books, using math quotes in your daily instruction is one of many
      ways to get students excited about this oft-dreaded subject. Over the
      centuries, some of the world’s brightest minds have shared wise words that
      remain relevant to this day! We’ve compiled a list of our favorite quotes
      about math for teachers and students alike! Inspire, motivate, and humor
      your students with these math quotes!
    </p>
    <p className="px-4 py-2">
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. At, a nihil
      maxime ipsa iste possimus id eius dolorem, incidunt ad ullam sunt.
      Explicabo adipisci nostrum sequi rem ut ad minus. Asperiores, architecto
      molestias aspernatur expedita dicta dolores dignissimos, voluptate
      officiis ducimus ullam officia velit earum repudiandae sint atque deleniti
      recusandae perferendis nihil voluptatibus. Sequi sint aliquam nulla ullam
      rerum magni! Quibusdam fuga temporibus nemo blanditiis alias laborum dicta
      voluptate voluptates esse, perferendis provident nesciunt reiciendis
      accusantium laudantium magni aliquam! Quo, nihil distinctio asperiores
      quis ducimus eveniet temporibus perferendis. Ad, dignissimos?
    </p>
  </section>
);

export default Home;
