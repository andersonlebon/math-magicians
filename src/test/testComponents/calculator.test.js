import renderer from 'react-test-renderer';
import home from '../../components/home';

it('renders correctly', () => {
  const tree = renderer.create(<home />).toJSON();
  expect(tree).toMatchSnapshot();
});
