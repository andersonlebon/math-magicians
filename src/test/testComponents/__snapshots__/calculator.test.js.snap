// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="d-flex calculator  w-100 justify-content-around"
>
  <p
    c={true}
    className="py-5 my-5"
  >
    Let's do some math!!
  </p>
  <div
    className="container-cal "
  >
    <div
      className="result"
    >
      0
    </div>
    <ul
      className="cal-table"
    >
      <li
        aria-hidden="true"
        className="cal-item makefalse"
        onClick={[Function]}
      >
        AC
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        +/-
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        %
      </li>
      <li
        aria-hidden="true"
        className="cal-item bg-orange"
        onClick={[Function]}
      >
        รท
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        7
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        8
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        9
      </li>
      <li
        aria-hidden="true"
        className="cal-item bg-orange"
        onClick={[Function]}
      >
        x
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        4
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        5
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        6
      </li>
      <li
        aria-hidden="true"
        className="cal-item bg-orange"
        onClick={[Function]}
      >
        -
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        1
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        2
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        3
      </li>
      <li
        aria-hidden="true"
        className="cal-item bg-orange"
        onClick={[Function]}
      >
        +
      </li>
      <li
        aria-hidden="true"
        className="cal-item item-0"
        onClick={[Function]}
      >
        0
      </li>
      <li
        aria-hidden="true"
        className="cal-item"
        onClick={[Function]}
      >
        .
      </li>
      <li
        aria-hidden="true"
        className="cal-item bg-orange"
        onClick={[Function]}
      >
        =
      </li>
    </ul>
  </div>
</section>
`;
